_includes/v19.2/misc/session-vars.html

```diff
@@ -95,6 +95,18 @@
    <td>Yes</td>
   </tr>
 
+  <tr>
+   <td>
+    <code>enable_zig_zag_join</code>
+   </td>
+   <td>Indicates whether the <a href="cost-based-optimizer.html">cost-based optimizer</a> will plan certain queries using a zig-zag merge join algorithm, which searches for the desired intersection by jumping back and forth between the indexes based on the fact that they share a sorted order in their key suffix. </td>
```

Comment:
nit: it's not really their key suffix, it's like...the prefix after you remove any constant columns. there's probably a more succinct way to express that.

Maybe something like:
> ... after constraining indexes, they share an ordering.

LGTM besides that

Reply:
Updated to use that language (I think).  PTAL.